src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionRuleServiceImpl.java

```diff
@@ -43,4 +43,45 @@ public ActionRule createActionRule(final ActionRule actionRule) {
     actionRule.setId(UUID.randomUUID());
     return actionRuleRepo.saveAndFlush(actionRule);
   }
+
+  @Override
+  @Transactional(propagation = Propagation.REQUIRED, timeout = TRANSACTION_TIMEOUT)
+  public ActionRule updateActionRule(final ActionRule actionRule) {
+    final UUID actionRuleId = actionRule.getId();
+
+    ActionRule existingActionRule = actionRuleRepo.findById(actionRuleId);
+    if (existingActionRule != null) {
```

Comment:
Would it be better to do a check the other way around?
```
if (existingActionRule == null) {
  return null
}
```
This way we're not indenting the whole rest of the code here